### ISSUE TITLE
django-rest-framework-filters not compatible with django-filter 0.10.0

### ISSUE THREAD

**Issue by ronsuez** [philipn/django-rest-framework-filters#42]
Django==1.8.3
djangorestframework-filters==0.5.0
django-filter==0.10.0
python Version 2.7.6

I added the library to my INSTALLED APPS but when i try to run the app , it throws me the following error:

```
Traceback (most recent call last):
  File "manage.py", line 13, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/ronsuez/.virtualenvs/lims/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/Users/ronsuez/.virtualenvs/lims/lib/python2.7/site-packages/django/core/management/__init__.py", line 312, in execute
    django.setup()
  File "/Users/ronsuez/.virtualenvs/lims/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/ronsuez/.virtualenvs/lims/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/Users/ronsuez/.virtualenvs/lims/lib/python2.7/site-packages/django/apps/config.py", line 86, in create
    module = import_module(entry)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/ronsuez/.virtualenvs/lims/lib/python2.7/site-packages/rest_framework_filters/__init__.py", line 3, in <module>
    from .filterset import FilterSet
  File "/Users/ronsuez/.virtualenvs/lims/lib/python2.7/site-packages/rest_framework_filters/filterset.py", line 52, in <module>
    class FilterSet(six.with_metaclass(FilterSetMetaclass, filterset.FilterSet)):
  File "/Users/ronsuez/.virtualenvs/lims/lib/python2.7/site-packages/rest_framework_filters/filterset.py", line 58, in FilterSet
    'filter_class': filters.IsoDateTimeFilter,
AttributeError: 'module' object has no attribute 'IsoDateTimeFilter'
```

was searching in the [django-filter-docs](http://django-filter.readthedocs.org/en/latest/ref/filters.html#isodatetimefilter) and they have that filter defined , but the package cannot found it

**Comment by rpkilby** [comment]
Hi @ronsuez. `IsoDateTimeFilter` was added in the 0.11.0 version of django-filter. You'll need to update the package.

**Comment by rpkilby** [comment]
The setup script will need to be updated to reflect the minimum version requirement. Regardless, the end result is that django-filter>=0.11.0 is required.

**Comment by ronsuez** [comment]
working, thanks @rpkilby

**Comment by rpkilby** [comment]
Fixed by a3556ed42c76a29841f1c9dc77524396c7554eaa
